docs/src/pages/_app.js

```diff
@@ -35,52 +34,93 @@ const MyApp = ({ Component, pageProps }) => {
   const router = useRouter();
   const propTable = pageProps?.mdxDirSource?.[router?.query?.doc]?.docData;
 
+  const theme = {
+    color: {
+      primary: '#b4104b',
+      secondary: '#7f7f7f40',
+    },
+  };
+
   return (
-    <>
-      <Head>
-        <title>Cascara</title>
-        <meta
-          content="Espressive's Functional Design System"
-          key='description'
-          name='description'
-        />
-        <meta
-          content='https://cascara.design/cascara_meta.png'
-          key='image'
-          property='og:image'
+    <AdminStructure
+      header={
+        <AdminStructure.Header
+          logo={['images?.logo']}
+          title={"Espressive's Functional Design System"}
         />
-        <meta content='width=device-width, initial-scale=1.0' name='viewport' />
-      </Head>
-      <Admin
-        drawer={
-          propTable?.length > 0 && (
-            <Admin.Drawer>
-              {propTable.map((componentProps) => (
-                <AnimatePresence
-                  exitBeforeEnter
-                  key={router.query.mdx + componentProps}
+      }
+      nav={<Nav {...pageProps} />}
+      theme={theme}
+    >
+      <AdminStructure.Main>
+        <Component {...pageProps} />
+      </AdminStructure.Main>
+      <AdminStructure.Drawer>
+        <div style={{ padding: '2rem' }}>
```

Comment:
@lordpixel this padding I do not think is needed in our new Drawer, so it might also be able to go away.

I think this is looking good so far.

That aside... I think we might want to remove the motion stuff for now with the animations. I think that might be something we include inside of our structure components themselves at a later date.

Good work so far!

Reply:
Nice, these changes have been apploied, thank you @brianespinosa